math/pi/montecarlopi.go

```diff
@@ -0,0 +1,27 @@
+// montecarlopi.go
+// description: Calculating pi by the Monte Carlo method
+// details:
+// implementation of Monte Carlo Algorithm for the calculating of Pi - [Monte Carlo method](https://en.wikipedia.org/wiki/Monte_Carlo_method)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see montecarlopi_test.go
+
+package pi
+
+import (
+	"math/rand"
+	"time"
+)
+
+func MonteCarloPi(n int) float64 {
```

Comment:
Try to be descriptive when writing code for someone else (for example this repository) 😄 I learnt it the hard way 😅.

```suggestion
func MonteCarloPi(randomPoints int) float64 {
```

or

```suggestion
func MonteCarloPi(dots int) float64 {
```

Reply:
Done